src/components/SshKeys/AddSshKey.js

```diff
@@ -16,7 +16,7 @@ const AddSshKey = ({me: { id, email }}) => {
     setValues({...values, [name]: value});
   }
 
-  const isFormValid = values.sshKeyName !== '' && !values.sshKey.includes('\n') &&
+  const isFormValid = values.sshKeyName !== '' &&
   (
     values.sshKey.trim().startsWith('ssh-rsa') ||
```

Comment:
actually, we should just introduce an `sshKey` value here - we're already running trim multiple times in this line. By the time we get to line 85, there's no reason why we should run trim on this string _again_. (I know this isn't part of this review, I just noticed it - good place to remove extraneous calls to trim)

Reply:
Created a new [issue](https://github.com/uselagoon/lagoon-ui/issues/64) to clean up the validation and add feedback in the UI.